go.mod

```diff
@@ -154,3 +154,5 @@ replace github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210
 replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 
 replace github.com/cosmos/cosmos-sdk/db => ./db
+
+replace github.com/tendermint/tendermint => github.com/yihuang/tendermint v0.35.0-rc1.0.20220217100838-fcd4744551f5
```

Comment:
https://github.com/tendermint/tendermint/pull/7837
update after merged in tendermint.

Reply:
is it okay we wait to merge this till 0.34.16 comes out, it should be this week

Reply:
https://github.com/tendermint/tendermint/blob/v0.34.16/CHANGELOG.md#v0.34.16

0.34.16 is released, want to update the pr then we can merge it

Reply:
it's already using tendermint `v0.35.x` branch, the next 0.35.x version is not released yet.

Reply:
0.35.2 is released.